[WC-1084] Remount video player on url change to not break history (#1403)
mendix/widgets-resources: packages/pluggableWidgets/video-player-web/CHANGELOG.md

## [Unreleased]

> - We fixed the issue when after changing URL property of Video Player, app navigation is get broken. (Ticket #143982)

Reviewer: ```suggestion
- We fixed the issue that when changing the URL property of the Video Player, app navigation is broken. (Ticket #143982)
```
